packages/remark-parse/lib/locate/escape.js

```diff
@@ -6,9 +6,7 @@
  * @fileoverview Locate an escape.
  */
 
-'use strict';
-
-module.exports = locate;
+export default locate;
 
 function locate(value, fromIndex) {
```

Comment:
for cases like this you can place the export keyword directly on locate, e.g.

`export default function locate()`